Library/Formula/boot.rb

```diff
@@ -0,0 +1,18 @@
+require "formula"
+
+class Boot < Formula
+  homepage "http://boot-clj.com"
+  version "2.0.0-rc4"
+  url "https://github.com/boot-clj/boot/releases/download/2.0.0-rc4/boot.sh",
```

Comment:
We should use https://github.com/boot-clj/boot/archive/2.0.0-rc4.tar.gz and install just the file we need.